src/@types/common.ts

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 import { JSXElementConstructor } from "react";
 
 // Based on https://stackoverflow.com/a/53229857/3532235
+// eslint-disable-next-line @typescript-eslint/type-annotation-spacing
```

Comment:
why is this needed now but wasn't before?

Reply:
Ah, an artifact of trying to do too many things... It's needed on a different branch which changes lint settings. I'll remove it here.